### PR TITLE
remove the option to disable the memory view from the settings

### PR DESCRIPTION
**Pull request by devoncarew**
- remove the option to disable the memory view from the settings

@terrylucas

### CODE REVIEW

File: src/io/flutter/settings/FlutterSettings.java [hunk 1]
```diff
@@ -32,8 +30,6 @@
   private static final String legacyTrackWidgetCreationKey = "io.flutter.trackWidgetCreation";
   private static final String disableTrackWidgetCreationKey = "io.flutter.disableTrackWidgetCreation";
   private static final String useFlutterLogView = "io.flutter.useLogView";
-  private static final String memoryProfilerKey = "io.flutter.memoryProfiler";
-  private static final String newBazelTestRunnerKey = "io.flutter.bazel.legacyTestBehavior";
```
Comment (terrylucas) [inline src/io/flutter/settings/FlutterSettings.java]: Something else that's outside of memory profiler?

File: src/io/flutter/view/FlutterPerfView.java [hunk 2]
```diff
@@ -207,7 +206,7 @@ private void addPerformanceViewContent(FlutterApp app, ToolWindow toolWindow) {
 
       addPerformanceTab(runnerTabs, app, toolWindow, true);
 
-      if (FlutterUtils.isAndroidStudio() && !FlutterSettings.getInstance().isMemoryProfilerDisabled()) {
```
Comment (terrylucas) [inline src/io/flutter/view/FlutterPerfView.java]: Should we silently remove the memory tab? Or would you like to wait until we either launch DevTool automatically or a link on how to use DevTools for profiling?

File: src/io/flutter/settings/FlutterSettings.java (hunk 1 above)
Comment (devoncarew) [reply src/io/flutter/settings/FlutterSettings.java]: yup, this was an unused field (from code removed by a previous PR)

File: src/io/flutter/view/FlutterPerfView.java (hunk 2 above)
Comment (devoncarew) [reply src/io/flutter/view/FlutterPerfView.java]: We should improve the way we launch devtools slightly. Perhaps open an issue to remove the memory view for the next release?

**Pull request merged**